packages/expo-cli/src/commands/build/AndroidBuilder.ts

```diff
@@ -195,9 +195,9 @@ See https://docs.expo.io/versions/latest/distribution/building-standalone-apps/#
         // read the keystore
         const keystoreData = await fs.readFile(keystorePath);
 
-        const credentials = {
+        const credentials: AndroidCredentials.Keystore = {
           keystore: keystoreData.toString('base64'),
-          keystoreAlias,
+          keyAlias: keystoreAlias,
```

Comment:
is this fix for apiv1 or v2 ?

Current version of expo-cli have default api set to v1 for know, but in PR description you mentioned that this is fix for apiv2? 

If this is fix for apiv1 then both `keyAlias` and `keystoreAlias` should still be supported in apiv1, because otherwise old clients will stop working. If this is intentional we should deprecate them.

If this is fix for apiv2 why default api is still on apiv1?

Reply:
This is a fix for apiv2.

I am keeping apiv2 default until the teams feature is supported on the backend.

Reply:
I updated the testing to clarify that it should be done with `EXPO_NEXT_API=true`.

Reply:
> 
> If this is fix for apiv1 then both `keyAlias` and `keystoreAlias` should still be supported in apiv1, because otherwise old clients will stop working. If this is intentional we should deprecate them.
> 

The object with renamed keys is only sent to an apiv2 endpoint, so we don't need to support `keystoreAlias`: `api.putAsync('credentials/android/keystore/${experienceName}'`